main.tf

```diff
@@ -46,7 +46,7 @@ resource "aws_vpc" "ttomczyk_vpc" {
 
 resource "aws_internet_gateway" "ttomczyk_aws_igw" {
   vpc_id = "aws_vpc.ttomczyk_vpc.id"
-  
+
   tags = {
     Name = "ttomczyk-aws-igw"
   }
```

Comment:
Nie mogę komentować niezmienionego kodu ale w subnetach cidrblocki trochę w nie tę stronę podzielone:
jak masz vpc na `10.0.0.0/16` to subnety tego samego rodzaju z reguły mają taką samą maskę (np. /20)
Czyli u Ciebie mogłoby być np
subnet-private-1: `10.0.0.0/20`
subnet-private-2: `10.0.16.0/20`
subnet-public-1: `10.0.32.0/22`
subnet-public-2: `10.0.36.0/22`
(prywatne mają /20 a publiczne /22 bo zazwyczaj potrzeba więcej adresów prywatnych niż publicznych)
tu taka mała ściąga:
https://www.aelius.com/njh/subnet_sheet.html